src/pacmod3_nodelet.cpp

```diff
@@ -47,7 +47,9 @@ void Pacmod3Nl::onInit()
   all_system_statuses_pub = nh_.advertise<pacmod3_msgs::AllSystemStatuses>("all_system_statuses", 20);
 
   global_rpt_pub = nh_.advertise<pacmod3_msgs::GlobalRpt>("global_rpt", 20);
-  component_rpt_pub = nh_.advertise<pacmod3_msgs::ComponentRpt>("component_rpt", 20);
+  component_rpt_00_pub = nh_.advertise<pacmod3_msgs::ComponentRpt>("component_rpt_00", 20);
```

Comment:
Can you add these new topics to the README documentation?

Reply:
yep!